docs/03_Development/01_Extending_Guide/04_Extending_Rule_Actions.md

```diff
@@ -59,7 +63,7 @@ final class CustomActionType extends AbstractType
 With configuration, comes a Javascript file as well:
 
 ```javascript
-//AppBundle/Resources/public/pimcore/js/custom_action.js
+//web/bundles/app/pimcore/js/custom_action.js
```

Comment:
thats not right, the file needs to sit inside the AppBundle and will be symlinked/copied via ```bin/console assets:install```